src/slams/credibilist/TBM_prob_conversion.h

```diff
@@ -0,0 +1,23 @@
+#ifndef SLAM_CTOR_SLAM_VINY_TBM_PROB_CONVERSION_H
+#define SLAM_CTOR_SLAM_VINY_TBM_PROB_CONVERSION_H
+
+#include "../../core/maps/transferable_belief_model.h"
+#include "../../core/states/state_data.h"
+
+// TBM ---> Occupancy
+Occupancy TBM_to_O(const TBM& tbm) {
+  return Occupancy{ tbm.occupied() + 0.5 * tbm.unknown(), 1.0 };
```

Comment:
Should the updated TBM_to_Occ conversion be used? Just wondering.

Reply:
Following your practice, I believe that should be put in a separate pull request?

Reply:
Agreed.